includes/jsdev/udf.js

```diff
@@ -0,0 +1,156 @@
+/**
```

Comment:
@lmajano :  I've pushed changes which should take care of these and add documentation to the JS methods. If I've missed anything from a coding standards standpoint, please let me know.